### DISCUSSION TITLE
23.11.16 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#68] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - PR 올리기
- 내일은 어떤 일을 하나요?
    - 발표 준비
    - 우재님 코드 리뷰

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 음악 저장 기능 동작 확인
  - PR 요청
- 내일은 어떤 일을 하나요?
  - 발표 준비(Spring Batch)
  - 민주님 코드 리뷰

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 리팩토링 및 PR 리뷰
* 내일은 어떤 일을 하나요?
  * 발표 준비
